include/library/syscalls.h

```diff
@@ -19,14 +19,15 @@ int process_fork();
 void process_exec(const char *path, const char **argv, int argc);
 int process_self();
 int process_parent();
-int process_kill( unsigned int pid );
```

Comment:
In the future, avoid making cosmetic changes in an unrelated area, that makes the code harder to merge.  (Or someone else's later commit harder to merge.)  Better to do formatting in a completely separate PR.